Tools/jit/_targets.py

```diff
@@ -507,6 +507,7 @@ def get_target(host: str) -> _COFF | _ELF | _MachO:
             # On aarch64 Linux, intrinsics were being emitted and this flag
             # was required to disable them.
             "-mno-outline-atomics",
+            "-fplt",
```

Comment:
This is needed otherwise trampolines are not generated at all and everything is referenced via the GOT. if you prefer, we can have this as separate PR.

Reply:
Yeah, this seems like this slows Linux down. Let's drop it.

Reply:
Done, removed.